src/revlanguage/datatypes/basic/RealPos.cpp

```diff
@@ -25,7 +25,7 @@ RealPos::RealPos( RevBayesCore::TypedDagNode<double> *x ) : Real( x )
     
     if ( x->getValue() < 0.0 )
```

Comment:
I'm wondering under what conditions an error like this one would be thrown, because the constructors for `RealPos` (unlike those for `Probability`) are not directly accessible to the user (i.e., through a Rev function). Can we maybe expand the test suite to include `ReaPos` as well?

Reply:
I'm not sure if command-line input can trigger this or not, but its probably still good to have a check here.